.gitlab-ci.yml

```
@@ -36,7 +36,7 @@ execute_plaster:
     - cp -r src/gdcdictionary/schemas/* gdcdictionary/$DICT_VERSION
     - git clone git@github.com:NCI-GDC/gdcdatamodel2.git
     - cd gdcdatamodel2
-    - git checkout "$CI_COMMIT_REF_NAME" || git checkout -b "$CI_COMMIT_REF_NAME"
+    - git checkout "gitlab_build_$CI_COMMIT_REF_NAME" || git checkout -b "gitlab_build_$CI_COMMIT_REF_NAME"
```

I sorta like a `/` vs `_` but at the end of the day up to you.
```suggestion
    - git checkout "gitlab_build/$CI_COMMIT_REF_NAME" || git checkout -b "gitlab_build/$CI_COMMIT_REF_NAME"
```